src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icon - Teilschild.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "􀙨.pdf",
```

Comment:
Github Issue or crazy name?

Reply:
It's a private character (u+100668), these are undefined, for letting you create your own character

Reply:
Renamed the icon files to make them more readable